sdk/core/azure-core/test/ut/transport_adapter_base_test.hpp

```diff
@@ -26,36 +26,13 @@ namespace Azure { namespace Core { namespace Test {
 
   struct AzureSdkHttpbinServer final
   {
-    inline static std::string Get()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + "/get";
-    }
-    inline static std::string Headers()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + "/headers";
-    }
-    inline static std::string WithPort()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + ":443/get";
-    }
-    inline static std::string Put()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + "/put";
-    }
-    inline static std::string Delete()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + "/delete";
-    }
-    inline static std::string Patch()
-    {
-      return std::string(_detail::AzureSdkHttpbinServerSchema) + "://"
-          + std::string(_detail::AzureSdkHttpbinServer) + "/patch";
-    }
+    inline static std::string Get() { return Schema() + "://" + Host() + "/get"; }
+    inline static std::string Headers() { return Schema() + "://" + Host() + "/headers"; }
+    inline static std::string WithPort() { return Schema() + "://" + Host() + ":443/get"; }
```

Comment:
```suggestion
    inline static std::string GetWithPort() { return Schema() + "://" + Host() + ":443/get"; }
```

Reply:
I like the change, but it's bigger than this one line.